src/pages/woong/Login/Login_woong.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import './Login.scss';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+
+function Login() {
+  const navigate = useNavigate();
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+  const condition = id.includes('@') && pw.length >= 5;
+
+  const goToList = () => {
+    condition === true && navigate('/woong/list');
```

Comment:
```suggestion
    goodCondition && navigate('/woong/list');
```
만약에 === true 부분을 줄이고 싶으시다면, semantic 차원에서 condition 이름을 goodCondition 으로 변경하고 === true를 생략하셔두 될 거 같애요!

Reply:
오 조언 감사드립니다 좋은 방법같아요